File: loop.java
public class Main
{
	public static void main(String[] args) {
		int a=0, b=3, c=3;
		long start = System.currentTimeMillis(), end;
		for ( int i=0; i<100000001; i++ ){
            		a += b*2 + c - i;
		}
		end = System.currentTimeMillis();
		System.out.println(a + " = a, " + (end-start) + " ms, " + (end-start)/100000000 + "speed iteration/ms" );
		
	}
}

File: loop.py
import datetime


a, b, c = 0, 3, 3
start = datetime.datetime.now()
for i in range(100000000):
	a += b*2 + c - i
end = datetime.datetime.now()
print(f'a = {a}, {(end-start).microseconds * 1000} ms, {((end-start).microseconds * 1000)/100000000} speed iteration/ms')